File: beecrowd/1000-1099/1040-1049/1046.py
entry = input().split()
entry = [eval(i) for i in entry]

time = 24
if entry[0] < entry[1]:
    time = entry[1] - entry[0]
if entry[0] > entry[1]:
    time = time - entry[0] + entry[1]
print(f"O JOGO DUROU {time} HORA(S)")

File: C99/1000-1099/1040-1049/1046.c
#include <stdio.h>
#include <math.h>

int main()
{
    int n1, n2;
    scanf("%d %d", &n1, &n2);

    if (n1 > n2)
    {
        n2 += 24;
        n1 = abs(n1 - n2);
        printf("O JOGO DUROU %d HORA(S)\n", n1);
    }
    else if (n1 == n2)
    {
        printf("O JOGO DUROU 24 HORA(S)\n");
    }
    else
    {
        n2 = abs(n2 - n1);
        printf("O JOGO DUROU %d HORA(S)\n", n2);
    }
    return 0;
}